docs/HowTo/Monitor/Metrics.md

```diff
@@ -0,0 +1,95 @@
+---
+description: Monitoring and metrics
+---
+
+Enable the [Prometheus](https://prometheus.io/) monitoring and alerting service for
+Teku metrics using the [`--metrics-enabled`](../../Reference/CLI/CLI-Syntax.md#metrics-enabled)
+option.
+
+## Install Prometheus
+
+To use Prometheus with Teku, install the
+[Prometheus main component](https://prometheus.io/download/). On MacOS, install with
+[Homebrew](https://formulae.brew.sh/formula/prometheus):
+
+ ```bash
+ brew install prometheus
+```
+
+!!! tip
+
+    You can also install:
+
+    * Exporters and send system metrics to Prometheus to monitor non-Teku specific items such as
+      disk usage and CPU usage.
+    * Other Prometheus components, such as the Alert Manager. Additional configuration is not
+      required for these components because Prometheus handles and analyzes data directly from the
+      feed.
+
+## Setting up and running Prometheus with Teku
+
+To configure Prometheus and run with Teku:
+
+1. Configure Prometheus to poll Teku. For example, add the following yaml fragment to the
+   `scrape_configs` block of the `prometheus.yml` file:
+
+    ```yml tab="Example"
+    global:
+      scrape_interval: 15s
+    scrape_configs:
+      - job_name: "prometheus"
+        static_configs:
+        - targets: ["localhost:9090"]
+      - job_name: "teku-dev"
+        scrape_timeout: 10s
+        metrics_path: /metrics
+        scheme: http
+        static_configs:
+        - targets: ["localhost:8008"]
+    ```
+
+1. Start Teku with the
+   [`--metrics-enabled`](../../Reference/CLI/CLI-Syntax.md#metrics-enabled) and
+   [`--metrics-categories`](../../Reference/CLI/CLI-Syntax.md#metrics-categories) options. To start a
+   node for testing with metrics enabled:
+
+     ```bash
+     teku --eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd \
+     --eth1-endpoint=http://localhost:8545 --validators-key-file=validator_keys \
+     --p2p-port=9000 --rest-api-enabled=true --rest-api-docs-enabled=true \
+     --metrics-enabled=true --metrics-categories=BEACON,PROCESS,LIBP2P,JVM,NETWORK,PROCESS
```

Comment:
valid values for metrics-categories: BEACON,LIBP2P,NETWORK,EVENTBUS

Reply:
ignore me as per conversation this morning

Reply:
This list has now beed updated. See https://github.com/PegaSysEng/teku/issues/1619